src/components/Dropdown/styles/_styles.scss

```diff
@@ -0,0 +1,35 @@
+.dropdown__btn {
+	 @include cx-btn--size(34px);
```

Comment:
You should define new style for the button but re-use the existing ones. Please remove this style and use buttons styles in the component directly.